global.go

```diff
@@ -151,18 +151,22 @@ func (gm *globalManager) sendHits(hits map[string]*RateLimitReq) {
 		}
 	}
 
+	var wg syncutil.WaitGroup
 	// Send the rate limit requests to their respective owning peers.
 	for _, p := range peerRequests {
-		ctx, cancel := context.WithTimeout(context.Background(), gm.conf.GlobalTimeout)
-		_, err := p.client.GetPeerRateLimits(ctx, &p.req)
-		cancel()
-
-		if err != nil {
-			gm.log.WithError(err).
-				Errorf("error sending global hits to '%s'", p.client.Info().GRPCAddress)
-			continue
-		}
+		p := p
+		wg.Go(func() {
```

Comment:
Instead of `syncutil.WaitGroup`, switch to `syncutil.FanOut()` so you can set a concurrency limit to avoid scaling issues.

Reply:
I set a limit of 100, I wasn't sure what number to set

Reply:
100 is ok.  And on second thought, can we make it configurable?  This is a performance tuning setting that could be critical in high throughput environments.